lib/puppet/type/jboss_deploy.rb

```diff
@@ -25,7 +25,12 @@
     desc "Indicate that server is in domain mode"
     defaultto true
   end
-  
+
```

Comment:
!fix Move those line from common params like controller, runasdomain. Thay are in most types and should be stacked for readability. See: https://github.com/coi-gov-pl/puppet-jboss/wiki/JBoss-module-standard-metaparameters